data/src/main/java/camp/nextstep/edu/github/GitHubInfoResponse.kt

```diff
@@ -0,0 +1,27 @@
+package camp.nextstep.edu.github
+
+import com.google.gson.annotations.SerializedName
+
+data class GitHubInfoResponse(
+    @SerializedName("full_name")
+    val fullName: String,
+
+    @SerializedName("description")
+    val description: String
+) {
+    fun fromGitHubInfoModel(): GitHubInfoModel {
```

Comment:
```suggestion
    fun toDomainModel(): GitHubInfoModel {
```

GitHubInfoResponse를 GitHubInfoModel로 변경하는 것이기 때문에 to가 더 어울리는거같네요!